### ISSUE TITLE
Trying to use an exported webgmeX (or its json) as a seed fails

### ISSUE THREAD

**Issue by finger563** [webgme/webgme#943]
*Importing* the `webgmeX` succeeds, but when using it as a seed I get the following error:

```
2016-04-14T03:38:49.619Z - error: [gme:server:worker:simpleworker:pid_16340:WorkerFunctions] seeding [samples.webgmeX] failed with error Error: unknown file seed [samples.webgmeX]
    at _findSeedFilename (/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/src/server/worker/workerrequests.js:421:33)
    at _getSeedFromFile (/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/src/server/worker/workerrequests.js:462:16)
    at /home/jeb/Repositories/webgme-rosmod/node_modules/webgme/src/server/worker/workerrequests.js:620:28
    at _fulfilled (/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/node_modules/q/q.js:796:13)
    at /home/jeb/Repositories/webgme-rosmod/node_modules/webgme/node_modules/q/q.js:604:44
    at runSingle (/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/node_modules/q/q.js:137:13)
    at flush (/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/node_modules/q/q.js:125:13)
    at nextTickCallbackWith0Args (node.js:420:9)
```
When trying to use the contained json as a seed I get this error:
```
2016-04-14T03:37:56.699Z - error: uncaughtException: Cannot read property 'guid' of undefined 
{ date: 'Wed Apr 13 2016 20:37:56 GMT-0700 (PDT)',
  process: 
   { pid: 16276,
     uid: 0,
     gid: 0,
     cwd: '/home/jeb/Repositories/webgme-rosmod',
     execPath: '/usr/bin/node',
     version: 'v4.4.0',
     argv: 
      [ '/usr/bin/node',
        '/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/src/server/worker/simpleworker.js' ],
     memoryUsage: { rss: 67170304, heapTotal: 44372576, heapUsed: 33960336 } },
  os: 
   { loadavg: [ 0.0703125, 0.0791015625, 0.11376953125 ],
     uptime: 20531 },
  trace: 
   [ { column: 64,
       file: '/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/src/common/core/users/serialization.js',
       function: 'Object.importLibrary [as import]',
       line: 1357,
       method: 'importLibrary [as import]',
       native: false },
     { column: 37,
       file: '/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/src/server/worker/workerrequests.js',
       function: null,
       line: 558,
       method: null,
       native: false },
     { column: 17,
       file: '/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/src/common/storage/storageclasses/editorstorage.js',
       function: null,
       line: 160,
       method: null,
       native: false },
     { column: 30,
       file: '/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/src/common/storage/socketio/websocket.js',
       function: '',
       line: 34,
       method: null,
       native: false },
     { column: 9,
       file: '/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/node_modules/socket.io-client/lib/socket.js',
       function: 'Socket.onack',
       line: 305,
       method: 'onack',
       native: false },
     { column: 12,
       file: '/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/node_modules/socket.io-client/lib/socket.js',
       function: 'Socket.onpacket',
       line: 229,
       method: 'onpacket',
       native: false },
     { column: 15,
       file: '/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/node_modules/socket.io-client/node_modules/component-bind/index.js',
       function: '',
       line: 21,
       method: null,
       native: false },
     { column: 20,
       file: '/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/node_modules/socket.io-client/node_modules/component-emitter/index.js',
       function: 'Manager.Emitter.emit',
       line: 131,
       method: 'Emitter.emit',
       native: false },
     { column: 8,
       file: '/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/node_modules/socket.io-client/lib/manager.js',
       function: 'Manager.ondecoded',
       line: 333,
       method: 'ondecoded',
       native: false },
     { column: 15,
       file: '/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/node_modules/socket.io-client/node_modules/component-bind/index.js',
       function: '',
       line: 21,
       method: null,
       native: false } ],
  stack: 
   [ 'TypeError: Cannot read property \'guid\' of undefined',
     '    at Object.importLibrary [as import] (/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/src/common/core/users/serialization.js:1357:64)',
     '    at /home/jeb/Repositories/webgme-rosmod/node_modules/webgme/src/server/worker/workerrequests.js:558:37',
     '    at /home/jeb/Repositories/webgme-rosmod/node_modules/webgme/src/common/storage/storageclasses/editorstorage.js:160:17',
     '    at Socket.<anonymous> (/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/src/common/storage/socketio/websocket.js:34:30)',
     '    at Socket.onack (/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/node_modules/socket.io-client/lib/socket.js:305:9)',
     '    at Socket.onpacket (/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/node_modules/socket.io-client/lib/socket.js:229:12)',
     '    at Manager.<anonymous> (/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/node_modules/socket.io-client/node_modules/component-bind/index.js:21:15)',
     '    at Manager.Emitter.emit (/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/node_modules/socket.io-client/node_modules/component-emitter/index.js:131:20)',
     '    at Manager.ondecoded (/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/node_modules/socket.io-client/lib/manager.js:333:8)',
     '    at Decoder.<anonymous> (/home/jeb/Repositories/webgme-rosmod/node_modules/webgme/node_modules/socket.io-client/node_modules/component-bind/index.js:21:15)' ] }
```

**Comment by kecso** [comment]
Yes.
1.7.0 is a transitional release.
The new project packages cannot be used as seeds yet.
Until the next release, if you wish to create new seeds, please use the ExportImport plugin.

Please let me know if that work for you!

**Comment by finger563** [comment]
Thanks, that does indeed work!

**Comment by kecso** [comment]
We reopen the case as it seems to be a better path for the transition to start allow seeding from the new package format as soon as possible. So the 'fix' will come with 1.7.1
